packages/manager/apps/dedicated/client/app/dedicated/server/translations/Messages_de_DE.json

```diff
@@ -1038,7 +1038,7 @@
   "server_ola_active": "Aktiv",
   "server_ola_inactive": "Inaktiv",
   "server_ola_activate": "OLA aktivieren",
-  "server_ola_configure": "OLA konfigurieren",
+  "server_ola_configure": "Die „private aggregation“ konfigurieren",
```

Comment:
Hmm 
```suggestion
  "server_ola_configure": "Die private aggregation konfigurieren",
```

Reply:
maybe because "private aggregation" is not translated :/

Reply:
Ok my bad, I was not aware of German style of commas x)